test/test_tree_2d_euler.jl

```diff
@@ -396,6 +396,33 @@ end
     end
 end
 
+@trixi_testset "elixir_euler_blast_wave_sc_subcell.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blast_wave_sc_subcell.jl"),
+                        l2=[
+                            0.3517507570120483,
+                            0.19252291020146015,
+                            0.19249751956580294,
+                            0.618717827188004,
+                        ],
+                        linf=[
+                            1.6699566795772216,
+                            1.3608007992899402,
+                            1.361864507190922,
+                            2.44022884092527,
+                        ],
+                        tspan=(0.0, 0.5),
+                        initial_refinement_level=4,
```

Comment:
This seems like a non-trivial test. Have you checked that it is within the test duration bounds (less than 10s on your computer after compilation)?

Reply:
Actually, it is very fast. The testset itself takes about 2 seconds. And the elixir takes only 100ms after compilation.